agent/engine/docker_task_engine.go

```diff
@@ -2065,7 +2065,9 @@ func (engine *DockerTaskEngine) stopContainer(task *apitask.Task, container *api
 	// Before attempting to stop any container, send drain signal for Appnet Agent to start draining connections
 	// (if not already in progress).
 	if task.IsServiceConnectEnabled() && !task.IsServiceConnectConnectionDraining() {
-		if err := engine.appnetClient.DrainInboundConnections(task.GetServiceConnectRuntimeConfig()); err != nil {
+		adminSocketPath := task.GetServiceConnectRuntimeConfig().AdminSocketPath
+		drainRequest := task.GetServiceConnectRuntimeConfig().DrainRequest
```

Comment:
Can you combine these two statement to only make one call?

Reply:
Sure.